### ISSUE TITLE
Fluent-Bit Issue: [msgpack2json] unknown msgpack type 1651862367

### ISSUE THREAD

**Issue by wengyao04** [fluent/fluent-bit#5540]
## Bug Report

**Describe the bug**

Hi, we use fluent bit `1.8.12`, kubernetes `1.20` without migrating to containerD runtime (still use docker). We find msgpack2json warning recently
```
{"log":"[2022/06/03 19:05:52] [ warn] [msgpack2json] unknown msgpack type 1651862367\n","stream":"stderr","time":"2022-06-03T19:05:52.757914081Z"}
```

But we don't know what causes this issue ? How could I debug this problem ?

our config is
```
apiVersion: v1
data:
  filter.conf: |
    [FILTER]
      Name          record_modifier
      Match         kube.*
      Allowlist_key time
      Allowlist_key log
      Allowlist_key stream
      Allowlist_key _p
  fluent-bit.conf: |
    [SERVICE]
      Flush           1
      Log_Level       info
      Daemon          off
      HTTP_Server     On
      HTTP_Listen     0.0.0.0
      HTTP_Port       2020
      storage.path    /var/log/fluent_bit_logging_storage
      storage.sync    normal
      storage.metrics On
    @INCLUDE input-kubernetes.conf
    @INCLUDE filter.conf
    @INCLUDE output.conf
  input-kubernetes.conf: |
    [INPUT]
      Name             tail
      Tag              kube.*
      Path             /var/log/containers/*.log
      multiline.parser docker
      DB               /var/log/flb_kube.db
      Buffer_Max_Size  1MB
      Mem_Buf_Limit    50MB
      Skip_Long_Lines  On
      Refresh_Interval 10
      storage.type     filesystem
  output.conf: |
    [OUTPUT]
      Name   file
      Format plain
      Match  kube.var.log.containers.*
      Path   /var/log/fluent_bit_containerd
kind: ConfigMap
metadata:
  labels:
    k8s-app: fluent-bit-logging
  name: fluent-bit-config
  namespace: fluent-bit-logging
```

**To Reproduce**
- Rubular link if applicable:
- Example log message if applicable:
```
{"log":"YOUR LOG MESSAGE HERE","stream":"stdout","time":"2018-06-11T14:37:30.681701731Z"}
```
- Steps to reproduce the problem:

**Expected behavior**

**Screenshots**

**Your Environment**

* Version used:
* Configuration:
* Environment name and version (e.g. Kubernetes? What version?):
* Server type and version:
* Operating System and version:
* Filters and plugins:

**Additional context**

**Comment by tarruda** [comment]
> But we don't know what causes this issue ? How could I debug this problem ?

It would help if you could run a debug version of fluent-bit with valgrind. Here's an example:

```
valgrind --log-file=valgrind.log fluent-bit -c [path_to_config]
```

Then attach `valgrind.log`

**Comment by wengyao04** [comment]
We run fluent-bit as daemonset on kubernetes. To use valgrind, do we need to install it from init container ?

**Comment by patrick-stephens** [comment]
I think this was asked in Slack as well but the gist is no, the `-debug` containers already contain valgrind. Additionally the production containers have no shell or package manager as they're based on Distroless ones.

However, I would also use the latest 1.9.4 containers to verify this is not a fixed bug - or at least the last of the 1.8 releases, i.e. 1.8.15.

**Comment by nokute78** [comment]
Note: In this pipeline, only out_file converts msgpack to json.
https://docs.fluentbit.io/manual/pipeline/outputs/file#plain-format

1651862367 = 0x62756B5F = "buk_" in ASCII.
It may be a part of "kube".
